tests/basics/fun_callstardblstar.py

```diff
@@ -37,3 +37,6 @@ def f2(*args, **kwargs):
 
 # case where *args is not a tuple/list and takes up most of the memory allocated for **kwargs
 f2(*iter(range(100)), **{str(i): i for i in range(100)})
+
+# regression test - was hitting assert(args2_len < args2_alloc)
```

Comment:
I don't think it's good to reference assertions in the code that may one day be removed/changed.  Better to instead describe the case that this is testing, namely that the size of the unknown iterables is small enough to fit within existing allocated array, but the `4, 5` after the iterables then overflow the array.

Reply:
fixe